### ISSUE TITLE
[Proposal] Логичный нерф репульса

### ISSUE THREAD

**Issue by QWxiZ** [TauCetiStation/TauCetiClassic#12197]
#### Подробное описание проблемы
Репульс - крайне надоедливая вещь.
Есть две возможности с ним справиться - невнимательность мага (чаще всего - подскок у него пинга или пролаг в игре, потому что скорость перезарядки довольно быстрая и невнимательность обычно прощается использованием скачка) или использование трюка с флешкой (человек с секундной флешкой подходит к магу вплотную, взрывает её, второй человек заканчивает магу жизнь).
Первый вариант - более рандомный и крайне маловероятный, если у мага нормальное интернет-соединение и есть опыт в игре; что, по сути, делает его непобедимым, если никто не пытается использовать второй вариант.
Второй вариант же в условиях ролевой игры на станции выглядит как-то уныло и неуместно: подбегать к другому человеку с активированной гранатой в руке, чтобы повалить и себя и его звучит нелогично и даже немного по-арбузерски (хоть это и не арбуз).
#### Что должно было произойти
Есть контра. 

Я предлагаю сделать так, чтобы магбутсы могли защищать от репульса.
Это логично: примагничивать ноги к полу это, по сути, работа магбутсов.
До этого не тяжело додуматься со стороны меты.
Они замедляют офицера, так что это не мега-ультимативное решение.

Дополнительный вариант баланcа: сделать так, чтобы более тяжелая броня имела на репульс воздействие. То есть, скажем, если человек носит тяжелую риотку и два щита в руках - его почти не откинет, потому что он переносит на себе 10 тысяч килограмм.

**Comment by simb11** [comment]
в игре существует такой реагент как синаптизин, оный позволяет встать почти сразу после каста репульса
это же буквально контра репульса

**Comment by ViDLiathan** [comment]
Пушечка то улетит

**Comment by Slavik2001** [comment]
Нет. Она же из рук выпадет и всё, если она уже лежит, то тогда улетит. И я щас вспомнил что на Парадизах есть спец имплант который фиксирует кисть чтобы то что ты держишь не могло выпасть в принципе. Только отделение конечности помогало

**Comment by Oz1Es** [comment]
про броню не согласен, а про магбутсы норм

**Comment by pro100kotah** [comment]
Твой вариант с флешкой контрится обычным скачком, который можно кастовать из любого состояния. Даже из стана.

**Comment by QWxiZ** [comment]
> Твой вариант с флешкой контрится обычным скачком, который можно кастовать из любого состояния. Даже из стана.

ну он работает если подловить после джаунта

**Comment by Deahaka** [comment]
А если не делать то, что в этом пропосале написано, то что будет? Дойдет ли до выпила мага как концепта нафиг? Если да, то не вижу смысла что-то делать, пусть предлагают как поменять режим чтобы было интереснее. А пока пусть маги будут неуловимыми и бессмертными (хаха)

**Comment by Naroke** [comment]
> Нет. Она же из рук выпадет и всё, если она уже лежит, то тогда улетит. И я щас вспомнил что на Парадизах есть спец имплант который фиксирует кисть чтобы то что ты держишь не могло выпасть в принципе. Только отделение конечности помогало

Блин, вот было бы круто, если бы на тау добавили аугменты. И роботехам дополнительная работа, и хирурги скучать не будут, и экипажу будет проще жить

**Comment by BlankCringe** [comment]
Я предлагаю автору выпить свои таблетки от шизы, и не нерфить самую говняную роль в игре, репульс это один из буквально нескольких полезных спеллов мага, которые делают больше чем щекотка.
